src/norm/sqlite.nim

```diff
@@ -354,11 +354,50 @@ proc update*[T: Model](dbConn; obj: var T) =
 
   dbConn.exec(sql qry, row)
 
+
 proc update*[T: Model](dbConn; objs: var openArray[T]) =
   ## Update rows for each `Model`_ instance in open array.
+  type BulkUpdateValues = Table[string, seq[tuple[id: int64; val: DbValue]]]
+
+  func genBulkUpdateQuery(tableName: string; values: BulkUpdateValues): tuple[qry: string; values: seq[DbValue]] =
```

Comment:
Is that a func definition inside `update`? Please make it a separate func definition in the outer scope.